js/editor.js

```diff
@@ -40,11 +40,12 @@ jQuery( function( $ ) {
 		if ( nextEditor.length ) {
 			cacheTranslationHelpersForARow( nextEditor );
 		}
-
-		if ( chatgpt_review_enabled && $gp_comment_feedback_settings.openai_key && $gp_editor_options.can_approve && ( 'waiting' === translation_status || 'fuzzy' === translation_status ) ) {
-			fetchOpenAIReviewResponse( rowId, tr, false );
+		if ( ! $gp_comment_feedback_settings.openai_key || ! $gp_editor_options.can_approve || ( 'waiting' !== translation_status && 'fuzzy' !== translation_status ) ) {
+			tr.find( '.details-chatgpt' ).hide();
+		} else if ( ! chatgpt_review_enabled ) {
+			tr.find( '.openai-review' ).hide();
```

Comment:
Does this work? I thought we need to close the details block, so shouldn't this rather be something like this?
```suggestion
			tr.find( '.details-chatgpt' ).attr( 'open', null );
```

Reply:
The check is simpler. We only need to hide the "ChatGPT Review" (main element) in some situations and call to `fetchOpenAIReviewResponse` if the `chatgpt_review_enabled` is true.